portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -93,4 +97,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the homepage.
     response.sendRedirect("/index.html");
   }
+
+  /**
+   * Calculates sentiment score of a comment's text and returns the score as a float. 
+   */
+  public float calculateSentimentScore(String text) throws IOException {
```

Comment:
Nit: since `calculateSentimentScore` is only used in this class, is it preferred to use a private function? I checked the style guide and didn't find a rule related to this. (Need a second opinion)

Reply:
Agreed, a safe rule of thumb is to default to private for all class members and only increase visibility when necessary.